### PR TITLE
(#735) Fix sensu-server-enterprise Vagrant VM

### LINKED ISSUE #735: Vagrant ports for sensu-enterprise-server are off
Currently one cannot test the API when using sensu-enterprise-server from the host. I believe that the redirected ports should be different in the Vagrantfile.

### PR DESCRIPTION
**Pull request by jeffmccune**
Without this patch the sensu-enterprise server is quickly crashing with a JVM
out of memory error when started inside the sensu-server-enterprise VM.

This patch addresses the problem by bumping up the allocated RAM for the VM
from 512m to 768m while also bringing down the JVM heap size from 2GB to 256m.

The API has been tested with the following commands, with no other VM's
running:

```bash
vagrant up sensu-server-enterprise
curl -s http://admin:secret@127.0.0.1:4567/clients \
  | jq -r 'map(.name | split(".") | .[0]) | sort | .[]'
```

Resolves #735

### CODE REVIEW

File: tests/sensu-server-enterprise.pp [hunk 1]
```diff
@@ -1,5 +1,14 @@
 node 'sensu-server' {
 
+  Ini_setting {
```
Comment (ghoneycutt) [inline tests/sensu-server-enterprise.pp]: please add that curl command to test in the comments here.

File: Vagrantfile [hunk 2]
```diff
@@ -25,7 +25,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.define "sensu-server", primary: true, autostart: true do |server|
     server.vm.box = "centos/7"
     server.vm.hostname = 'sensu-server.example.com'
-    server.vm.network :private_network, ip: "192.168.56.10"
+    server.vm.network :private_network, ip: ENV['VAGRANT_SENSU_SERVER_IP'] || '192.168.56.10'
```
Comment (ghoneycutt) [inline Vagrantfile]: Could you please document this in the Vagrantfile

File: tests/sensu-server-enterprise.pp (hunk 1 above)
Comment (jeffmccune) [inline tests/sensu-server-enterprise.pp]: Added!

File: Vagrantfile (hunk 2 above)
Comment (jeffmccune) [inline Vagrantfile]: Sure, thing, documentation added to the top of the Vagrantfile.  I also renamed the variables and consolidated to `ENV['ALTERNATE_IP']` for clarity.

**Comment by ghoneycutt** [comment]
Released in v2.22.0

**Pull request merged**